package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mathjax-node-svg2png",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
Thanks for the vote. I think I'll remove the `38`, make a minor release, then add EXTOPX properly and make a major release.